# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xfig/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- xfig.3.2.4/u_fonts.h.vanilla	2005-09-02 17:50:09.000000000 +0000
+++ xfig.3.2.4/u_fonts.h	2005-09-02 17:51:13.000000000 +0000
@@ -32,9 +32,6 @@
 
 extern int		psfontnum();
 extern int		latexfontnum();
-extern struct _xfstruct	x_fontinfo[], x_backup_fontinfo[];
-extern struct _fstruct	ps_fontinfo[];
-extern struct _fstruct	latex_fontinfo[];
 
 /* element of linked list for each font
    The head of list is for the different font NAMES,
@@ -62,4 +59,9 @@
 };
 
 int		x_fontnum();
+
+extern struct _xfstruct x_fontinfo[], x_backup_fontinfo[];
+extern struct _fstruct  ps_fontinfo[];
+extern struct _fstruct  latex_fontinfo[];
+
 #endif /* U_FONTS_H */
